### PR TITLE
docs: update roadmap for koor operator

### PR DESCRIPTION
**Pull request by zalsader**
Add details for koor operator, replace placeholders

### CODE REVIEW

File: docs/development/ksd-product-roadmap.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ The Koor Storage Distribution includes features that work on top of Rook and Cep
 
 | Feature   |   State   | Description                     |
 | --------- |   :---:   | ------------------------------- |
-| something | [0.2.0 released](https://github.com/koor-tech/koor-operator/releases/tag/koor-operator-0.2.0) | describe something it does already |
-| version update notifications | in development |  a brief statement of what this will do |
-| automated Ceph upgrades | future | a brief statement of what this will do |
+| Installing rook | [0.2.0 released](https://github.com/koor-tech/koor-operator/releases/tag/koor-operator-0.2.0) | Installs rook and checks if the cluster resources match recommendations |
```
Comment (galexrt) [inline docs/development/ksd-product-roadmap.md:17]: Should we replace `Rook` with `Koor` here? As it technically will/should deploy our fork of Rook.

File: docs/development/ksd-product-roadmap.md [hunk 2]
```diff
@@ -14,9 +14,9 @@ The Koor Storage Distribution includes features that work on top of Rook and Cep
...
+| Version update notifications | in development |  sends a notification whenever a new version of rook or ceph is released and ready for use |
```
Comment (galexrt) [inline docs/development/ksd-product-roadmap.md:18]:
```suggestion
| Version update notifications | in development | Sends a notification whenever a new version of rook or ceph is released and ready for use |
```

Should we mention the CRD as the "notification target" or is that too technical for the product roadmap?

File: docs/development/ksd-product-roadmap.md (hunk 1 above)
Comment (dave-at-koor) [reply docs/development/ksd-product-roadmap.md:17]: I guess that distinction will matter more in the future. Good enough for now.

File: docs/development/ksd-product-roadmap.md (hunk 2 above)
Comment (dave-at-koor) [reply docs/development/ksd-product-roadmap.md:18]: Not too technical. On the other hand, that could be part of a detailed description of each feature, outside of this list.

**Review by dave-at-koor — APPROVED** [review]

**Pull request merged**